src/dawn/Optimizer/PassDataLocalityMetric.h

```diff
@@ -41,6 +52,14 @@ class PassDataLocalityMetric : public Pass {
   HardwareConfig config_;
 };
 
+std::pair<int, int> computeReadWriteAccessesMetric(StencilInstantiation* instantiation,
```

Comment:
This was here previously... I think it returns total number of Reads and total number of Writes

Reply:
sorry, I didnt get it. Why not using here ReadWriteAccumulator better than std::pair? like that we know what each member of the pair means, so is more readable.
Also why do we pass StencilInstantiation* instead of reference of shared_ptr ?
And my final comment here, can we add documentation (doxygen) what the function does?